packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -155,6 +160,36 @@ type BaseBoxVisualProps = MakeObjectResponsive<
 
 type BoxVisualProps = MakeObjectResponsive<{
   backgroundColor: BackgroundColorString<'surface'>;
+  /**
+   * **Warning**
```

Comment:
Adding warning until we pick up #1234

Reply:
That issue only exists for our repo no? so should we have this warning? On react native projects it will anyway be `undefined`

Reply:
Also should we move `background*` and `border*` props to some `CommonVisualPropsType` kind of object? then we won't have to add these types twice.

I had initially created `BoxVisualProps` and `BaseBoxVisualProps` because visual props deferred between Box and BaseBox. They had different types (E.g. backgroundColor for Box has different type from backgroundColor for BaseBox). This is not the case anymore for backgroundImage, border props.

Reply:
Yeah makes sense, let me clean it up a bit

Reply:
I've refactored it to make it a bit cleaner but the `PickCSSByPlatform` is not working as intended anyway. Not even for the other types like `placeSelf` that were there already. See if you can take a quick look at it otherwise we can raise an issue for it for fixing `PickCSSByPlatform`